File: include/PolyWorksSDK/samples/plugins/common/CSharp/CSharpLineScanFramework/Commands/StrUtil.cs
// =================================================================================================
// String utilities.
//
// Copyright © InnovMetric Logiciels Inc. 2009 All Rights Reserved
// -------------------------------------------------------------------------------------------------
namespace IM
{
    public static class StrUtil
    {
        public static string StrNonNull(string str_)
        {
            if(str_ == null)
            {
                return str_ = string.Empty;
            }
            return str_;
        }
    }
}

File: include/PolyWorksSDK/samples/plugins/common/VisualBasic.NET/VBNet3DSceneRendering/COM_SDK/Util/StrUtil.vb
' =================================================================================================
' String utilities.
'
' Copyright © InnovMetric Logiciels Inc. 2012 All Rights Reserved
' -------------------------------------------------------------------------------------------------
Namespace VBNet3DSceneRendering

    Public Class StrUtil

        Public Shared Function StrNonNull(ByVal str_ As String) As String

            If str_ Is Nothing Then

                Return str_ = String.Empty

            End If

            Return str_

        End Function

    End Class

End Namespace




